cogs/draw_image_generation.py

```diff
@@ -6,20 +6,19 @@
 
 import discord
 from PIL import Image
-from discord.ext import commands
+from pycord.multicog import add_to_group

```

Comment:
Also, what does multicog actually do?

Reply:
The decorator adds the slash commands to a list group named from the string. Then in apply iterates over them and sets the slash command guild_id to the slash group guild_id, removes it from pending commands and adds the slash group as the parent of the slash command, also applies the original cog, or the slash group cog as fallback.